WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -381,6 +386,18 @@ private void newPostSetup() {
         mShortcutUtils.reportShortcutUsed(Shortcut.CREATE_NEW_POST);
     }
 
+    private void createPostEditorAnalyticsSessionTracker() {
+        if (mPostEditorAnalyticsSession == null) {
+            mPostEditorAnalyticsSession = new PostEditorAnalyticsSession(
+                    mShowGutenbergEditor ? Editor.GUTENBERG : Editor.CLASSIC,
+                    mPost, mSite);
+            // set initial outcome for published posts
```

Comment:
Hey Mario, can you expand on why do we need to set an initial outcome for published posts? I'm not sure I follow. Thanks!

Reply:
I think this is a leftover from my initial attempt to follow the same structure as in iOS where the `forceOutcome()` logic exists. Now that we only do `setOutcome()` without trying to interpret anything  (just following along state changes in the various points where `setOutcome()` is called) We can get rid of it

Reply:
removed in f0d26407be